Remove most content.

jdswinbank opened the pull request:
There was a lot of material duplicated between this page and the Stingray
documentation. This was doubly confusing, because that information was (in
genereal) only relevant to Stingray itself, and not to HENDRICS and DAVE which
are also covered by this page.

This PR makes this page into just a “landing page” for the StingraySoftware
organization. It provides links to all the associated projects; their pages
are assumed to provide all the information that readers will need, so it isn't
duplicated here.

(The one thing that wasn't already in the Stingray docs was the list of talks,
so I'm going to PR an addition of that there.)

jdswinbank committed:
Remove most content.

There was a lot of material duplicated between this page and the Stingray
documentation. This was doubly confusing, because that information was (in
genereal) only relevant to Stingray itself, and not to HENDRICS and DAVE which
are also covered by this page.

This PR makes this page into just a “landing page” for the StingraySoftware
organization. It provides links to all the associated projects; their pages
are assumed to provide all the information that readers will need, so it isn't
duplicated here.

(The one thing that wasn't already in the Stingray docs was the list of talks,
so I'm going to PR an addition of that there.)

matteobachetti commented on index.html:
```diff
@@ -29,54 +28,15 @@ <h2 id="project_tagline">Next-generation spectral-timing software</h2>
-    <li><a alt="Stingray PyAstro16" href="https://speakerdeck.com/abigailstev/stingray-pyastro16">Python in Astronomy 2016</a></li>
-</ul></p>
-
-<h4><a id="acknowledgements" class="anchor" href="#acknowledgements" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Acknowledgements</h4>
```
I think the acknowledgements should stay, either here or maybe in the Stingray repository

jdswinbank replied:
They're listed (albeit without the graphics) in the Stingray docs: https://stingraysoftware.github.io/stingray/acknowledgements.html. @dhuppenkothen indicated she thought that was the best place for them. Should I add the logos there as well?

matteobachetti replied:
Ah, ok. I think that Stingraysoftware benefited as a whole from PyCharm's free licenses and from GSoC, so it's maybe good to have them here too. @dhuppenkothen, @abigailStev, what do you think?

matteobachetti reviewed (changes requested):
Thanks a lot @jdswinbank, the edits make sense. Minor change requested.

jdswinbank committed:
Remove unused images

jdswinbank committed:
Tweak wording in link

jdswinbank committed:
Move image to page header.

A huge image with a small amount of text looks kinda weird, but I think this
header effect is quite neat.

jdswinbank committed:
Add last-updated date in footer

This seems more useful than another link to GitHub.

jdswinbank committed:
Add acknowledgements

jdswinbank commented:
Hey @matteobachetti — how's this?

jdswinbank committed:
Update website URL

jdswinbank committed:
Use new URL to core package

matteobachetti commented:
Looks good!

matteobachetti commented:
@jdswinbank if you're done, I'll merge

jdswinbank commented:
Go for it! 👍
